vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/format/NoticeEventFormatter.kt

```diff
@@ -95,7 +95,7 @@ class NoticeEventFormatter @Inject constructor(private val activeSessionDataSour
 
     private fun formatRoomPowerLevels(event: Event, disambiguatedDisplayName: String): CharSequence? {
         val powerLevelsContent: PowerLevelsContent = event.getClearContent().toModel() ?: return null
-        val previousPowerLevelsContent: PowerLevelsContent = event.prevContent.toModel() ?: return null
+        val previousPowerLevelsContent: PowerLevelsContent = event.resolvedPrevContent().toModel() ?: return null
```

Comment:
I was wondering why we were not using `resolvedPrevContent()` everywhere. Not clear in the spec about which part of the Json has to be taken into account. So Ok for these change!

Reply:
You can't have both in same time, I think it's mostly in unsignedData now...

Reply:
IIRC some event have both `prev_content` and `unsigned_data`, but ok anyway